app/models/like.rb

```diff
@@ -0,0 +1,11 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
```

Comment:
- [ ] As I already told you that the author is the alias of the user then it would be better if you make the belongs_to relationship of the like with the author 👍 After making the changes it would look like the below ⬇️

```ruby
belongs_to :author, class_name: 'User'
belongs_to :post
```